ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -365,7 +416,7 @@ impl RawData for RawDataServerWrapper {
                             end_version = end_of_batch_version,
                             batch_size = current_batch_size,
                             tps = (tps_calculator.avg() * 1000.0) as u64,
-                            "[Indexer Data] Sending batch."
+                            "Batch stats."
```

Comment:
Can you add more description to the log message? What statistics is being logged in the context of the data service?

Reply:
I can't comment on line 423, but that one needs more dimensions too